src/com/lftechnology/java/training/sanish/stemleaf/StemLeafDemo.java

```diff
@@ -0,0 +1,99 @@
+package com.lftechnology.java.training.sanish.stemleaf;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class StemLeafDemo {
+	private static final Logger LOGGER = Logger.getLogger(StemLeafDemo.class
+			.getName());
+	private static ArrayList<Integer> numberPoints = new ArrayList<Integer>();
+	private static ArrayList<StemLeaf> stemLeafs = new ArrayList<StemLeaf>();
+
+	private StemLeafDemo() {
+	};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)